### DISCUSSION TITLE
NPM stopped working - SOLVED

### DISCUSSION THREAD

**Discussion by cdebackere** [NginxProxyManager/nginx-proxy-manager#2466] [Q&A]
Edit: solved
For some reason another container got activated taking use of some ports that also npm needed, hence not possible. That's probably what code 500 means.

Sorry to bother ..
Christof
-------------
NMP stopped working from its docker container.

In my docker portainer portal I see:
Status: Stopped for 15 minutes with exit code 128

And when I try to start, then I get 'Request failed with status code 500'
This is my config:

<html>
<body>
<div class="row ng-scope" style="box-sizing: border-box; --tw-border-spacing-x:0; --tw-border-spacing-y:0; --tw-translate-x:0; --tw-translate-y:0; --tw-rotate:0; --tw-skew-x:0; --tw-skew-y:0; --tw-scale-x:1; --tw-scale-y:1; --tw-pan-x: ; --tw-pan-y: ; --tw-pinch-zoom: ; --tw-scroll-snap-strictness:proximity; --tw-ordinal: ; --tw-slashed-zero: ; --tw-numeric-figure: ; --tw-numeric-spacing: ; --tw-numeric-fraction: ; --tw-ring-inset: ; --tw-ring-offset-width:0px; --tw-ring-offset-color:#fff; --tw-ring-color:rgba(59,130,246,0.5); --tw-ring-offset-shadow:0 0 #0000; --tw-ring-shadow:0 0 #0000; --tw-shadow:0 0 #0000; --tw-shadow-colored:0 0 #0000; --tw-blur: ; --tw-brightness: ; --tw-contrast: ; --tw-grayscale: ; --tw-hue-rotate: ; --tw-invert: ; --tw-saturate: ; --tw-sepia: ; --tw-drop-shadow: ; --tw-backdrop-blur: ; --tw-backdrop-brightness: ; --tw-backdrop-contrast: ; --tw-backdrop-grayscale: ; --tw-backdrop-hue-rotate: ; --tw-backdrop-invert: ; --tw-backdrop-opacity: ; --tw-backdrop-saturate: ; --tw-backdrop-sepia: ; margin-top: 0px; margin-right: 0px !important; margin-bottom: 0px; margin-left: 0px !important; padding: 0px; color: rgb(51, 51, 51); font-family: Inter; font-size: 14px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(243, 243, 243); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><div class="col-lg-12 col-md-12 col-xs-12" style="box-sizing: border-box; --tw-border-spacing-x:0; --tw-border-spacing-y:0; --tw-translate-x:0; --tw-translate-y:0; --tw-rotate:0; --tw-skew-x:0; --tw-skew-y:0; --tw-scale-x:1; --tw-scale-y:1; --tw-pan-x: ; --tw-pan-y: ; --tw-pinch-zoom: ; --tw-scroll-snap-strictness:proximity; --tw-ordinal: ; --tw-slashed-zero: ; --tw-numeric-figure: ; --tw-numeric-spacing: ; --tw-numeric-fraction: ; --tw-ring-inset: ; --tw-ring-offset-width:0px; --tw-ring-offset-color:#fff; --tw-ring-color:rgba(59,130,246,0.5); --tw-ring-offset-shadow:0 0 #0000; --tw-ring-shadow:0 0 #0000; --tw-shadow:0 0 #0000; --tw-shadow-colored:0 0 #0000; --tw-blur: ; --tw-brightness: ; --tw-contrast: ; --tw-grayscale: ; --tw-hue-rotate: ; --tw-invert: ; --tw-saturate: ; --tw-sepia: ; --tw-drop-shadow: ; --tw-backdrop-blur: ; --tw-backdrop-brightness: ; --tw-backdrop-contrast: ; --tw-backdrop-grayscale: ; --tw-backdrop-hue-rotate: ; --tw-backdrop-invert: ; --tw-backdrop-opacity: ; --tw-backdrop-saturate: ; --tw-backdrop-sepia: ; margin: 0px 0px 15px; padding: 0px 15px; min-height: 1px; position: relative; float: left; width: 963px;"><rd-widget class="ng-isolate-scope" style="box-sizing: border-box; --tw-border-spacing-x:0; --tw-border-spacing-y:0; --tw-translate-x:0; --tw-translate-y:0; --tw-rotate:0; --tw-skew-x:0; --tw-skew-y:0; --tw-scale-x:1; --tw-scale-y:1; --tw-pan-x: ; --tw-pan-y: ; --tw-pinch-zoom: ; --tw-scroll-snap-strictness:proximity; --tw-ordinal: ; --tw-slashed-zero: ; --tw-numeric-figure: ; --tw-numeric-spacing: ; --tw-numeric-fraction: ; --tw-ring-inset: ; --tw-ring-offset-width:0px; --tw-ring-offset-color:#fff; --tw-ring-color:rgba(59,130,246,0.5); --tw-ring-offset-shadow:0 0 #0000; --tw-ring-shadow:0 0 #0000; --tw-shadow:0 0 #0000; --tw-shadow-colored:0 0 #0000; --tw-blur: ; --tw-brightness: ; --tw-contrast: ; --tw-grayscale: ; --tw-hue-rotate: ; --tw-invert: ; --tw-saturate: ; --tw-sepia: ; --tw-drop-shadow: ; --tw-backdrop-blur: ; --tw-backdrop-brightness: ; --tw-backdrop-contrast: ; --tw-backdrop-grayscale: ; --tw-backdrop-hue-rotate: ; --tw-backdrop-invert: ; --tw-backdrop-opacity: ; --tw-backdrop-saturate: ; --tw-backdrop-sepia: ; margin: 0px; padding: 0px;"><div class="widget" ng-transclude="" style="box-sizing: border-box; --tw-border-spacing-x:0; --tw-border-spacing-y:0; --tw-translate-x:0; --tw-translate-y:0; --tw-rotate:0; --tw-skew-x:0; --tw-skew-y:0; --tw-scale-x:1; --tw-scale-y:1; --tw-pan-x: ; --tw-pan-y: ; --tw-pinch-zoom: ; --tw-scroll-snap-strictness:proximity; --tw-ordinal: ; --tw-slashed-zero: ; --tw-numeric-figure: ; --tw-numeric-spacing: ; --tw-numeric-fraction: ; --tw-ring-inset: ; --tw-ring-offset-width:0px; --tw-ring-offset-color:#fff; --tw-ring-color:rgba(59,130,246,0.5); --tw-ring-offset-shadow:0 0 #0000; --tw-ring-shadow:0 0 #0000; --tw-shadow:0 0 #0000; --tw-shadow-colored:0 0 #0000; --tw-blur: ; --tw-brightness: ; --tw-contrast: ; --tw-grayscale: ; --tw-hue-rotate: ; --tw-invert: ; --tw-saturate: ; --tw-sepia: ; --tw-drop-shadow: ; --tw-backdrop-blur: ; --tw-backdrop-brightness: ; --tw-backdrop-contrast: ; --tw-backdrop-grayscale: ; --tw-backdrop-hue-rotate: ; --tw-backdrop-invert: ; --tw-backdrop-opacity: ; --tw-backdrop-saturate: ; --tw-backdrop-sepia: ; margin: 0px; padding: 0px; background: var(--bg-widget-color); border: 1px solid var(--border-widget); border-radius: 8px; box-shadow: rgba(0, 0, 0, 0.05) 0px 1px 1px;"><rd-widget-body classes="no-padding" class="ng-scope ng-isolate-scope" style="box-sizing: border-box; --tw-border-spacing-x:0; --tw-border-spacing-y:0; --tw-translate-x:0; --tw-translate-y:0; --tw-rotate:0; --tw-skew-x:0; --tw-skew-y:0; --tw-scale-x:1; --tw-scale-y:1; --tw-pan-x: ; --tw-pan-y: ; --tw-pinch-zoom: ; --tw-scroll-snap-strictness:proximity; --tw-ordinal: ; --tw-slashed-zero: ; --tw-numeric-figure: ; --tw-numeric-spacing: ; --tw-numeric-fraction: ; --tw-ring-inset: ; --tw-ring-offset-width:0px; --tw-ring-offset-color:#fff; --tw-ring-color:rgba(59,130,246,0.5); --tw-ring-offset-shadow:0 0 #0000; --tw-ring-shadow:0 0 #0000; --tw-shadow:0 0 #0000; --tw-shadow-colored:0 0 #0000; --tw-blur: ; --tw-brightness: ; --tw-contrast: ; --tw-grayscale: ; --tw-hue-rotate: ; --tw-invert: ; --tw-saturate: ; --tw-sepia: ; --tw-drop-shadow: ; --tw-backdrop-blur: ; --tw-backdrop-brightness: ; --tw-backdrop-contrast: ; --tw-backdrop-grayscale: ; --tw-backdrop-hue-rotate: ; --tw-backdrop-invert: ; --tw-backdrop-opacity: ; --tw-backdrop-saturate: ; --tw-backdrop-sepia: ; margin: 0px; padding: 0px;"><div class="widget-body no-padding" ng-class="$ctrl.classes" style="box-sizing: border-box; --tw-border-spacing-x:0; --tw-border-spacing-y:0; --tw-translate-x:0; --tw-translate-y:0; --tw-rotate:0; --tw-skew-x:0; --tw-skew-y:0; --tw-scale-x:1; --tw-scale-y:1; --tw-pan-x: ; --tw-pan-y: ; --tw-pinch-zoom: ; --tw-scroll-snap-strictness:proximity; --tw-ordinal: ; --tw-slashed-zero: ; --tw-numeric-figure: ; --tw-numeric-spacing: ; --tw-numeric-fraction: ; --tw-ring-inset: ; --tw-ring-offset-width:0px; --tw-ring-offset-color:#fff; --tw-ring-color:rgba(59,130,246,0.5); --tw-ring-offset-shadow:0 0 #0000; --tw-ring-shadow:0 0 #0000; --tw-shadow:0 0 #0000; --tw-shadow-colored:0 0 #0000; --tw-blur: ; --tw-brightness: ; --tw-contrast: ; --tw-grayscale: ; --tw-hue-rotate: ; --tw-invert: ; --tw-saturate: ; --tw-sepia: ; --tw-drop-shadow: ; --tw-backdrop-blur: ; --tw-backdrop-brightness: ; --tw-backdrop-contrast: ; --tw-backdrop-grayscale: ; --tw-backdrop-hue-rotate: ; --tw-backdrop-invert: ; --tw-backdrop-opacity: ; --tw-backdrop-saturate: ; --tw-backdrop-sepia: ; margin: 0px; padding: 0px; border-radius: 8px;"><div ng-hide="$ctrl.loading" class="widget-content" ng-transclude="" style="box-sizing: border-box; --tw-border-spacing-x:0; --tw-border-spacing-y:0; --tw-translate-x:0; --tw-translate-y:0; --tw-rotate:0; --tw-skew-x:0; --tw-skew-y:0; --tw-scale-x:1; --tw-scale-y:1; --tw-pan-x: ; --tw-pan-y: ; --tw-pinch-zoom: ; --tw-scroll-snap-strictness:proximity; --tw-ordinal: ; --tw-slashed-zero: ; --tw-numeric-figure: ; --tw-numeric-spacing: ; --tw-numeric-fraction: ; --tw-ring-inset: ; --tw-ring-offset-width:0px; --tw-ring-offset-color:#fff; --tw-ring-color:rgba(59,130,246,0.5); --tw-ring-offset-shadow:0 0 #0000; --tw-ring-shadow:0 0 #0000; --tw-shadow:0 0 #0000; --tw-shadow-colored:0 0 #0000; --tw-blur: ; --tw-brightness: ; --tw-contrast: ; --tw-grayscale: ; --tw-hue-rotate: ; --tw-invert: ; --tw-saturate: ; --tw-sepia: ; --tw-drop-shadow: ; --tw-backdrop-blur: ; --tw-backdrop-brightness: ; --tw-backdrop-contrast: ; --tw-backdrop-grayscale: ; --tw-backdrop-hue-rotate: ; --tw-backdrop-invert: ; --tw-backdrop-opacity: ; --tw-backdrop-saturate: ; --tw-backdrop-sepia: ; margin: 0px; padding: 0px;">

IMAGE | jc21/nginx-proxy-manager:latest@sha256:60a6ddeeaa79b8f0cda43b3ebcb789e453caff8a919070e930746dbb2cd24878
-- | --
CMD |  
ENTRYPOINT | /init
ENV | CROWDSEC_OPENRESTY_BOUNCER_VERSION0.1.7CURL_CA_BUNDLE/etc/ssl/certs/ca-certificates.crtNODE_ENVproductionNPM_BUILD_COMMITcce73beNPM_BUILD_DATE2022-03-31 05:46:34 UTCNPM_BUILD_VERSION2.9.18OPENRESTY_VERSION1.19.9.1PATH/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/binS6_BEHAVIOUR_IF_STAGE2_FAILS1S6_FIX_ATTRS_HIDDEN1SSL_CERT_FILE/etc/ssl/certs/ca-certificates.crtSUPPRESS_NO_CONFIG_WARNING1 | CROWDSEC_OPENRESTY_BOUNCER_VERSION | 0.1.7 | CURL_CA_BUNDLE | /etc/ssl/certs/ca-certificates.crt | NODE_ENV | production | NPM_BUILD_COMMIT | cce73be | NPM_BUILD_DATE | 2022-03-31 05:46:34 UTC | NPM_BUILD_VERSION | 2.9.18 | OPENRESTY_VERSION | 1.19.9.1 | PATH | /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin | S6_BEHAVIOUR_IF_STAGE2_FAILS | 1 | S6_FIX_ATTRS_HIDDEN | 1 | SSL_CERT_FILE | /etc/ssl/certs/ca-certificates.crt | SUPPRESS_NO_CONFIG_WARNING | 1
CROWDSEC_OPENRESTY_BOUNCER_VERSION | 0.1.7
CURL_CA_BUNDLE | /etc/ssl/certs/ca-certificates.crt
NODE_ENV | production
NPM_BUILD_COMMIT | cce73be
NPM_BUILD_DATE | 2022-03-31 05:46:34 UTC
NPM_BUILD_VERSION | 2.9.18
OPENRESTY_VERSION | 1.19.9.1
PATH | /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
S6_BEHAVIOUR_IF_STAGE2_FAILS | 1
S6_FIX_ATTRS_HIDDEN | 1
SSL_CERT_FILE | /etc/ssl/certs/ca-certificates.crt
SUPPRESS_NO_CONFIG_WARNING | 1
LABELS | com.docker.compose.config-hashb23f5d8df8b576b3bd52f3b220dc1d048f355d160355c6d89d7ea9959523d11acom.docker.compose.container-number1com.docker.compose.oneoffFalsecom.docker.compose.projectcdebackerecom.docker.compose.project.config_filesdocker-compose.ymlcom.docker.compose.project.working_dir/home/cdebackerecom.docker.compose.serviceappcom.docker.compose.version1.29.2maintainerJamie Curnow <jc@jc21.com>org.label-schema.cmddocker run --rm -ti jc21/nginx-proxy-manager:latestorg.label-schema.descriptionDocker container for managing Nginx proxy hosts with a simple, powerful interfaceorg.label-schema.licenseMITorg.label-schema.namenginx-proxy-managerorg.label-schema.schema-version1.0org.label-schema.urlhttps://github.com/jc21/nginx-proxy-managerorg.label-schema.vcs-urlhttps://github.com/jc21/nginx-proxy-manager.git | com.docker.compose.config-hash | b23f5d8df8b576b3bd52f3b220dc1d048f355d160355c6d89d7ea9959523d11a | com.docker.compose.container-number | 1 | com.docker.compose.oneoff | False | com.docker.compose.project | cdebackere | com.docker.compose.project.config_files | docker-compose.yml | com.docker.compose.project.working_dir | /home/cdebackere | com.docker.compose.service | app | com.docker.compose.version | 1.29.2 | maintainer | Jamie Curnow <jc@jc21.com> | org.label-schema.cmd | docker run --rm -ti jc21/nginx-proxy-manager:latest | org.label-schema.description | Docker container for managing Nginx proxy hosts with a simple, powerful interface | org.label-schema.license | MIT | org.label-schema.name | nginx-proxy-manager | org.label-schema.schema-version | 1.0 | org.label-schema.url | https://github.com/jc21/nginx-proxy-manager | org.label-schema.vcs-url | https://github.com/jc21/nginx-proxy-manager.git
com.docker.compose.config-hash | b23f5d8df8b576b3bd52f3b220dc1d048f355d160355c6d89d7ea9959523d11a
com.docker.compose.container-number | 1
com.docker.compose.oneoff | False
com.docker.compose.project | cdebackere
com.docker.compose.project.config_files | docker-compose.yml
com.docker.compose.project.working_dir | /home/cdebackere
com.docker.compose.service | app
com.docker.compose.version | 1.29.2
maintainer | Jamie Curnow <jc@jc21.com>
org.label-schema.cmd | docker run --rm -ti jc21/nginx-proxy-manager:latest
org.label-schema.description | Docker container for managing Nginx proxy hosts with a simple, powerful interface
org.label-schema.license | MIT
org.label-schema.name | nginx-proxy-manager
org.label-schema.schema-version | 1.0
org.label-schema.url | https://github.com/jc21/nginx-proxy-manager
org.label-schema.vcs-url | https://github.com/jc21/nginx-proxy-manager.git
RESTART POLICIES | Name           None           On Failure           Always           Unless Stopped         Update | Name | None           On Failure           Always           Unless Stopped | Update
Name | None           On Failure           Always           Unless Stopped | Update

</div></div></rd-widget-body></div></rd-widget></div></div>
</body>
</html>
IMAGE	[jc21/nginx-proxy-manager:latest@sha256:60a6ddeeaa79b8f0cda43b3ebcb789e453caff8a919070e930746dbb2cd24878](https://192.168.255.152:9443/#!/2/docker/images/sha256%3A60a6ddeeaa79b8f0cda43b3ebcb789e453caff8a919070e930746dbb2cd24878)
CMD	
ENTRYPOINT	/init
ENV	
CROWDSEC_OPENRESTY_BOUNCER_VERSION	0.1.7
CURL_CA_BUNDLE	/etc/ssl/certs/ca-certificates.crt
NODE_ENV	production
NPM_BUILD_COMMIT	cce73be
NPM_BUILD_DATE	2022-03-31 05:46:34 UTC
NPM_BUILD_VERSION	2.9.18
OPENRESTY_VERSION	1.19.9.1
PATH	/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
S6_BEHAVIOUR_IF_STAGE2_FAILS	1
S6_FIX_ATTRS_HIDDEN	1
SSL_CERT_FILE	/etc/ssl/certs/ca-certificates.crt
SUPPRESS_NO_CONFIG_WARNING	1
LABELS	
com.docker.compose.config-hash	b23f5d8df8b576b3bd52f3b220dc1d048f355d160355c6d89d7ea9959523d11a
com.docker.compose.container-number	1
com.docker.compose.oneoff	False
com.docker.compose.project	cdebackere
com.docker.compose.project.config_files	docker-compose.yml
com.docker.compose.project.working_dir	/home/cdebackere
com.docker.compose.service	app
com.docker.compose.version	1.29.2
maintainer	Jamie Curnow <jc@jc21.com>
org.label-schema.cmd	docker run --rm -ti jc21/nginx-proxy-manager:latest
org.label-schema.description	Docker container for managing Nginx proxy hosts with a simple, powerful interface
org.label-schema.license	MIT
org.label-schema.name	nginx-proxy-manager
org.label-schema.schema-version	1.0
org.label-schema.url	https://github.com/jc21/nginx-proxy-manager
org.label-schema.vcs-url	https://github.com/jc21/nginx-proxy-manager.git
RESTART POLICIES	
Name	
Unless Stopped
Volumes
Host/volume	Path in container
/home/cdebackere/data	/data
/home/cdebackere/letsencrypt	/etc/letsencrypt

**Comment by asder28** [comment]
good work
